app/controllers/collections_controller.rb

```diff
@@ -21,6 +22,7 @@ def new
 
   # GET /collections/1/edit
   def edit
+    @is_edit = true
```

Comment:
This `@is_edit` boolean seems off to me.

Perhaps this is the point that we should start diverting our show and edit partials and be more explicit about this.  I'm more than happy to talk through other solutions if it seems like we would be duplicating too much boilerplate templating code (although I'm not sure we know that now).